Dockerfile

```diff
@@ -18,4 +18,6 @@ RUN mix release --path /app-release
 FROM hexpm/elixir:1.18.2-erlang-26.2.5.9-alpine-3.19.7 as runtime
 
 COPY --from=elixir /app-release .
+
+HEALTHCHECK CMD ["bin/uncooked_gps", "rpc", "UncookedGps.Health.healthy?()"]
```

Comment:
note: you'll also need to add this to the configuration in OpenTofu; ECS requires you to specify it explicitly.

Reply:
https://github.com/mbta/terraform_modules/pull/226/files#diff-8634502043723872e23e75b61d109e8addd61f65854535c33a13b96a9345b454R29